docs/amq-streams.asciidoc

```diff
@@ -0,0 +1,177 @@
+│   ├── debezium-core-0.10.0.Beta1-redhat-00001.jar
+│   ├── debezium-ddl-parser-0.10.0.Beta1-redhat-00001.jar
+│   ├── mysql-binlog-connector-java-0.19.1.redhat-00002.jar
+│   └── mysql-connector-java-8.0.16.jar
+├── debezium-connector-postgres
+│   ├── CHANGELOG.md
+│   ├── CONTRIBUTE.md
+│   ├── COPYRIGHT.txt
+│   ├── LICENSE-3rd-PARTIES.txt
+│   ├── LICENSE.txt
+│   ├── README.md
+│   ├── debezium-connector-postgres-0.10.0.Beta1-redhat-00001.jar
+│   ├── debezium-core-0.10.0.Beta1-redhat-00001.jar
+│   ├── postgresql-42.2.5.jar
+│   └── protobuf-java-2.6.1.redhat-1.jar
+└── debezium-connector-sqlserver
+    ├── CHANGELOG.md
+    ├── CONTRIBUTE.md
+    ├── COPYRIGHT.txt
+    ├── LICENSE-3rd-PARTIES.txt
+    ├── LICENSE.txt
+    ├── README.md
+    ├── debezium-connector-sqlserver-0.10.0.Beta1-redhat-00001.jar
+    ├── debezium-core-0.10.0.Beta1-redhat-00001.jar
+    └── mssql-jdbc-7.2.2.jre8.jar
+----
+
+=== Deploying Kafka Connect with the Debezium Developer Preview
+
+With plugin directory structure ready, all we have to do now in order to deploy our Kafka Connect cluster is triggering the ```s2i``` build.
+
+[listing,subs="attributes",options="nowrap"]
+----
+# Check the name of our build config
+$ oc get buildconfigs
+NAME                                     TYPE     FROM     LATEST
+debezium-kafka-connect-cluster-connect   Source   Binary   2
+
+# Build and deploy Kafka Connect cluster with Debezium
+$ oc start-build debezium-kafka-connect-cluster-connect --from-dir=./plugins
+----
+
+After a while all parts should be up and running:
+[source%nowrap,bash]
+----
+oc get pods
+
+NAME                                                      READY     STATUS      RESTARTS   AGE
+debezium-kafka-cluster-entity-operator-7b6b9d4c5f-k7b92   3/3       Running     0          10m
+debezium-kafka-cluster-kafka-0                            2/2       Running     0          9m
+debezium-kafka-cluster-zookeeper-0                        2/2       Running     0          9m
+debezium-kafka-connect-cluster-connect-2-jw695            1/1       Running     0          1m
+debezium-kafka-connect-cluster-connect-2-deploy           0/1       Completed   0          3m
+strimzi-cluster-operator-97cd5cf7b-l58bq                  1/1       Running     0          11m
+----
+
+Alternatively, you can go to the "Pods" view of your OpenShift Web Console to confirm all pods are up and running:
+
+image::/images/openshift_amq_pods.png[width=771,align="center"]
+
+== Verifying the Deployment
+
+Next we are going to verify whether the deployment is correct by emulating the link:/docs/tutorial/[Debezium Tutorial] and following the steps in link"/docs/openshift/#verifying_the_deployment"[OpenShift Installation]
+
+[NOTE]
+====
+You will want to change the pod names to correspond with your AMQ Streams deployment.
+====
+
+== Questions and Resources
+If you got any questions or requests related to running Debezium on OpenShift,
```

Comment:
We should see what to link here. Either that new mailing list or some other channel. I'll clarify.

Reply:
Definitely, there is more place where links need to be added or updated.

Reply:
Ok, so here is how the sentence should look like:

> In case of any requests or questions related to running the Debezium Developer Preview with AMQ Streams on OpenShift, please let us know by sending an e-mail to debezium-cdc-preview@redhat.com.

Reply:
Done